visualization/app/codeCharta/ui/matchingFilesCounter/matchingFilesCounter.component.html

```diff
@@ -1,3 +1,7 @@
-<span class="box-rounded node-count">{{ $ctrl._viewModel.fileCount }}<i class="fa fa-file"></i></span>
-<span class="box-rounded node-count">{{ $ctrl._viewModel.flattenCount }}<i class="fa fa-eye-slash"></i></span>
-<span class="box-rounded node-count">{{ $ctrl._viewModel.excludeCount }}<i class="fa fa-ban"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.fileCount }}<i class="fa fa-file"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.flattenCount }}<i class="fa fa-eye-slash"></i></span>
+<span class="box-rounded node-count ">{{ $ctrl._viewModel.excludeCount }}<i class="fa fa-ban"></i></span>
```

Comment:
empty space at the end of the class name is still here.

Reply:
i misunderstood the request and inserted empty spaces instead, should be fixed